Remove std::binary_function, which is removed in C++17. Taken from
https://github.com/openbabel/openbabel/pull/2464

Index: scripts/openbabel-perl.i
--- scripts/openbabel-perl.i.orig
+++ scripts/openbabel-perl.i
@@ -191,13 +191,6 @@ CAST_GENERICDATA_TO(VirtualBond)
 
 %import <openbabel/chains.h>
 %import <openbabel/typer.h>
-
-// To avoid warning in oberror.h about "Nothing known about std::binary_function"
-namespace std { 
-        template <T1, T2, T3>
-        class binary_function {}; 
-}
-%template(Dummy) std::binary_function <const char *, const char *, bool>;
 %include <openbabel/plugin.h>
 
 // To avoid warning in oberror.h about "Nothing known about std::stringbuf"
